src/features/wallet/hooks/cosmos.ts

```diff
@@ -108,8 +108,8 @@ export function useCosmosTransactionFns(): ChainTransactionFns {
         // The fee param of 'auto' here stopped working for Neutron-based IBC transfers
         // It seems the signAndBroadcast method uses a default fee multiplier of 1.4
         // https://github.com/cosmos/cosmjs/blob/e819a1fc0e99a3e5320d8d6667a08d3b92e5e836/packages/stargate/src/signingstargateclient.ts#L115
-        // Bumping to 1.6 fixes the insufficient gas issue
-        result = await client.signAndBroadcast(chainContext.address, [tx.transaction], 1.6);
+        // A multiplier of 1.6 was insufficient for Celestia -> Neutron|Cosmos -> XXX transfers, but 2 worked.
+        result = await client.signAndBroadcast(chainContext.address, [tx.transaction], 2);
```

Comment:
Can you get this change merged to `main` as well please?

Reply:
yep! https://github.com/hyperlane-xyz/hyperlane-warp-ui-template/pull/310